src/main/webapp/style.css

```diff
@@ -101,15 +115,24 @@ html {
   background-color: #4c8bf5 !important;
 }
 
+#text {
+  margin: auto;
+  margin-top: 100px;
+  max-width: 45%;
+}
+
 #text-cont {
   padding-top: 25px;
   margin: auto;
   width: 50%;
 }
 
-@font-face {
-  font-family: 'Product Sans';
-  font-style: normal;
-  font-weight: 400;
-  src: url('./fonts/product-sans/ProductSansRegular.ttf') format('truetype');
+#view-btn-cont a {
```

Comment:
Can you group the selectors and add section comments? I think it would be helpful if we're only using 1 style sheet.

https://google.github.io/styleguide/htmlcssguide.html#Section_Comments

Reply:
Waiting for Cassandra and Fola

Reply:
We do not need it alphabetized, but can you please split this css into multiple files? You can pause this PR and get this fix out by branching off of master. Then get that approved and merged and then just sync this branch.